cmake/InstallLayout.cmake

```diff
@@ -215,17 +215,17 @@ set(CPACK_COMPONENT_SERVER-DEB_DEPENDS clients-deb)
 set(CPACK_COMPONENT_SERVER-TGZ_DEPENDS clients-tgz)
 set(CPACK_COMPONENT_SERVER-VERSIONED_DEPENDS clients-versioned)
 set(CPACK_RPM_SERVER-VERSIONED_PACKAGE_REQUIRES
-  "foundationdb-clients-${FDB_MAJOR}.${FDB_MINOR}.${FDB_PATCH} = ${FDB_MAJOR}.${FDB_MINOR}.${FDB_PATCH}")
+  "foundationdb${PROJECT_VERSION}-clients")
```

Comment:
The removal of `= X.Y.Z` works because there is only one version of the versioned clients package by this name?

Reply:
yes exactly -- you can install any version of `foundationdb6.3.20` :)